13/umbraco-ui-builder/getting-started/configuration.md

```diff
@@ -8,7 +8,7 @@ Umbraco UI Builder can be configured directly via the `AddUIBuilder` extension m
 
 ## AddUIBuilder
 
-To configure Umbraco UI Builder via the `AddUIBuilder` extension method, You can look in the `Program.cs` file in the root of your web project. From within this file, before the call to `AddComposers()` we can add our `AddUIBuilder` configuration.
+To configure Umbraco UI Builder via the `AddUIBuilder` extension method, You can look in the `Startup.cs` file in the root of your web project. From within this file, before the call to `AddComposers()` we can add our `AddUIBuilder` configuration.
```

Comment:
It's been a while since I've seen any activity here, so here's a suggestion to move this one along a bit.

Let me know what you think!
I'll merge it within the next week, if I haven't heard back from you 💪 

```suggestion
To configure Umbraco UI Builder via the `AddUIBuilder` extension method, You can look in the `Program.cs` file at the root of your web project. From within this file, before the call to `AddComposers()` we can add our `AddUIBuilder` configuration.

Alternatively, if your project is upgraded from earlier versions, you can add the configuration to the `Startup.cs` file.
```